File: edu-platform.web-master/edu-platform.web-master/src/services/instructor/response/InstructorResponse.ts
import {BaseResponse} from "../../BaseResponse";

export interface Instructor {
  id: number;
  familyName: string;
  firstName: string;
  email: string;
  yearOfBirth: number;
  provinceId: number;
  districtId: number;
  wardId: number;
  address: string;
  onlineCourseDeliver: boolean;
  inPersonCourseDeliver: boolean;
  status: string;
}

export interface InstructorResponse extends BaseResponse {
  data: Instructor;
}

File: edu-platform-master/edu-platform-master/src/main/java/edu/platform/domains/instructor/model/response/InstructorResponse.java
package edu.platform.domains.instructor.model.response;

import edu.platform.domains.instructor.constant.InstructGender;
import edu.platform.domains.instructor.constant.InstructorStatus;
import edu.platform.domains.instructor.constant.InstructorVerifyStatus;
import lombok.Data;

@Data
public class InstructorResponse {
    private Long id;
    private String familyName;
    private String firstName;
    private String email;
    private Integer yearOfBirth;
    private InstructGender gender;
    private String address;
    private Long provinceId;
    private Long districtId;
    private Long wardId;
    private Boolean onlineCourseDeliver;
    private Boolean inPersonCourseDeliver;
    private InstructorStatus status;
    private Boolean isVerified;
    private InstructorVerifyStatus verifyStatus;
}